### PR TITLE
Reduce webui_infos usage in request controller

### PR DESCRIPTION
**Pull request by coolo**
Move the important parts of webui_info into the controller, most work is still in webui_actions for now though

### CODE REVIEW

**Comment by coolo** [comment]
This is just prepwork for the real feature I want to implement - but in the light of small PRs, I stop here :)

**Comment by vpereira** [comment]
could this PR and the PR https://github.com/openSUSE/open-build-service/pull/7016 be merged in one PR?

**Comment by coolo** [comment]
I wasnt' aware of that one - @bgeuken could have mentioned it when I mentioned that I would like to hack on requests :)

@bgeuken, shall I merge?

**Comment by bgeuken** [comment]
> I wasnt' aware of that one - @bgeuken could have mentioned it when I mentioned that I would like to hack on requests :)

It didn't come to my mind:-) but I also didn't realize that you've been planning to refactor this code, since you said you want to work on the UI part [again](#6225). 
Anyhow, as you can see the PR / branch is some month old. So feel free to ignore it or fetch changes from there.

File: src/api/app/controllers/webui/request_controller.rb [hunk 1]
```diff
@@ -48,10 +48,26 @@ def add_reviewer
     redirect_to controller: :request, action: 'show', number: params[:number]
   end
 
-  def modify_review
-    review_params = params.slice(:comment, :by_user, :by_group, :by_project, :by_package)
-    request = BsRequest.find_by_number(params[:request_number])
+  def modify_review_set_request
+    review_params = params.slice(:comment, :by_user, :by_group, :by_project, :by_package, :review_id)
```
Comment (bgeuken) [inline src/api/app/controllers/webui/request_controller.rb]: If I understand correctly, the `by_*` parameters won't be needed anymore once the bento UI is dropped. I would add a comment for that.

Comment (coolo) [reply src/api/app/controllers/webui/request_controller.rb]: there is a TODO bento_only 2 lines below - but I extend it

**Pull request merged**